### PR TITLE
Fix font-weight normal handling; fixes #756

### LINKED ISSUE #756: bug in rule `font-weight-notation` in shorthand property `font`
Shorthand property `font` may any order property `font-style`, `font-weight`, `font-stretch`.
Example
```css
p.test {
    font: italic small-caps bold large serif;
}

p.test1 {
    font: small-caps italic bold large serif;
}

p.test2 {
    font: bold small-caps italic large serif;
}
...
```
But if i use and set `font-weight-notation` to `numeric`
```css
p {
    font: normal normal normal large serif;
}
```
I get error (it is normal):
```
2:11	⚠  Expected numeric font-weight notation (font-weight-notation) [stylelint]
```
I change code to:
```
p {
    font: normal normal 400 large serif;
}
```
But i get error:
```
2:11	⚠  Expected numeric font-weight notation (font-weight-notation) [stylelint]
```
Chrome opens all property to
```css
font-style: normal;
font-variant: normal;
font-weight: 400;
font-stretch: normal;
font-size: large;
line-height: normal;
font-family: serif;
```
I can change order, but get error all the same:
```css
p {
    font: 400 normal normal large serif;
}
```
```css
p {
    font: normal 400 normal large serif;
}
```

### PR DESCRIPTION
**Pull request by davidtheclark**

### CODE REVIEW

File: src/rules/font-weight-notation/index.js [hunk 1]
```diff
@@ -46,11 +48,20 @@ export default function (expectation, options) {
     })
 
     function checkFont(decl) {
+      const valueList = postcss.list.space(decl.value)
+      // We do not need to more carefully distinguish font-weight
+      // numbers from unitless line-heights because line-heights in
+      // `font` values need to be part of a font-size/line-height pair
+      const hasNumericFontWeight = valueList.some(isNumbery)
+      console.log(JSON.stringify(hasNumericFontWeight))
```
Comment (jeddy3) [inline src/rules/font-weight-notation/index.js]: console.log

File: src/rules/font-weight-notation/index.js [hunk 2]
```diff
@@ -46,11 +48,20 @@ export default function (expectation, options) {
...
+
       for (let value of postcss.list.space(decl.value)) {
-        // We do not need to more carefully distinguish font-weight
-        // numbers from unitless line-heights because line-heights in
-        // `font` values need to be part of a font-size/line-height pair
-        if (isNumbery(value) || NAMED_WEIGHTS.indexOf(value) !== -1) {
+        console.log(JSON.stringify(value))
```
Comment (jeddy3) [inline src/rules/font-weight-notation/index.js]: and here. Once that up stream eslint issue is resolved, we'll get `no-console` going here.

**Comment by jeddy3** [comment]
This looks good to me.

**Comment by jeddy3** [comment]
I just released btw. Sorry about that, I didn't realise you were working on this at the time. I got my times mixed up and I thought it was night in the states! When in fact it's the morning.

**Comment by ntwb** [comment]
*And lunchtime here down under* :koala:

**Pull request merged**